frontend/src/components/visual-editor/form/inputs/triggers/PatternInput.tsx

```diff
@@ -279,6 +274,7 @@ const PatternInput: FC<PatternInputProps> = ({
         {typeof value === "string" && patternType === "text" ? (
           <Input
             {...(getInputProps ? getInputProps(idx) : null)}
+            {...registerInput(t("message.text_is_required"), idx)}
```

Comment:
Hi @PranavBhatP, i'm not sure about this JSX instruction.
Is there a specific reason to add this for input text ?

Reply:
Yes, this was meant to catch empty text fields from being submitted, but it seems redundant, now that I have tested the functionality without this line of code. Will remove it in the following commit.

Reply:
Hi @PranavBhatP, thank you for your interactivity
I have found an issue with the current code version
Can you please test the scenario reproduced in the video bellow to double check if this use case is covered ?
<video src="https://github.com/user-attachments/assets/db63c354-96eb-4606-b0fa-9af0b9f187cc" />

Reply:
Hi, I've made changes addressing all the missing cases mentioned before, made enhancements to ensure error state is reflected in the different supported languages as well.

Reply:
Hi @PranavBhatP, i will test the changes and give you feedback :technologist: